File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 5 - C/q7.c
//Faça um algoritmo para ler o código e o preço de 15 produtos, calcular e escrever:
//a) o maior preço lido
//b) a média aritmética dos preços dos produtos
#include<stdio.h>
int main(){
	int i;
	float preco1, preco, maior, soma, media;
	printf("Preco: ");
	scanf("%f", &preco1);
	maior=preco1;
	soma=0;
	for(i=0; i<2; i++){
		printf("Preco: ");
		scanf("%f", &preco);
		if(preco>maior){
			maior=preco;
		}
        soma=soma+preco;
	}
	media=(preco1+soma)/3;
	printf("Maior preço: %f\nMedia: %f\n", maior, media);	
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 5 - Python/q7.py
#Faça um algoritmo para ler o código e o preço de 15 produtos, calcular e escrever:
#a) o maior preço lido
#b) a média aritmética dos preços dos produtos

cod=int(input("CODIGO: "))
preco1=float(input("PREÇO: "))
maior=preco1
soma=0
for i in range(2):
    cod=int(input("CODIGO: "))
    preco=float(input("PREÇO: "))
    if preco>maior:
        maior=preco
    soma=soma+preco
media=(preco1+soma)/3
print("Maior preço: ", maior, "\nMedia: ", media)
    
    
    
